tools/hmac/calculateHmacPayments.js

```diff
@@ -0,0 +1,73 @@
+// script to calculate the HMAC signature of Payments webhooks (where the signature is calculated considering
+// a subset of the fields in the payload - i.e. NotificationRequestItem object)
+//
+// Run with: `node calculateHmacPayments.js {hmacKey} {path to JSON file}
+//
+// cd tools/hmac
+// node calculateHmacPayments.js 11223344D785FBAE710E7F943F307971BB61B21281C98C9129B3D4018A57B2EB payload.json
+
+const fs = require('fs');
+const { hmacValidator } = require('@adyen/api-library');
+
+// Ensure correct arguments
+if (process.argv.length !== 4) {
+    console.error("Usage: node calculateHmacPayments.js <hmacKey> <payloadFile>");
+    process.exit(1);
+}
+
+const hmacKey = process.argv[2];
+const payloadFile = process.argv[3];
+
+// Check if file exists
+if (!fs.existsSync(payloadFile)) {
```

Comment:
nit: rename "payloadFile" to:
- If relative: `relativeFilePath`
- If absolute: `absoluteFilePath`

nit #2: add comment for the relativePath `// The relative path to the .json file`